pkg/backup/actions/csi/pvc_action.go

```diff
@@ -252,6 +254,23 @@ func (p *pvcBackupItemAction) Execute(
 		return nil, nil, "", nil, err
 	}
 
+	// Wait until VS associated VSC snapshot handle created before
+	// continue.we later require the vsc restore size
+	vsc, err := csi.WaitUntilVSCHandleIsReady(
```

Comment:
Not sure about this change.
This does make the logic more unified for both data mover and CSI cases; however, the CSI case already includes a check in the VolumeSnapshot BIA.
It is a bit redundant to also check in the PVC BIA.
https://github.com/vmware-tanzu/velero/blob/9500f0b82f1ec1df9ebe83c4f6896b1a32c99a38/pkg/backup/actions/csi/volumesnapshot_action.go#L102-L111